public/scripts/controllers/booksShowController.js

```diff
@@ -0,0 +1,49 @@
+angular
+	.module('bookApp')
+	.controller('BooksShowController', BooksShowController);
+
+BooksShowController.$inject = ['$http', '$routeParams', '$location'];
+function BooksShowController (  $http, 	 $routeParams,	 $location ) {
+	var vm = this;
+	var bookId = $routeParams.id;
+	vm.book;
+	vm.bookCopy; //Will use to edit input fields without changing the values displayed in the view until they are saved
+
+	$http({
+		method: 'GET',
+		url: 'https://super-crud.herokuapp.com/books/' + bookId,
+	}).then(function successCallback(response){
+		vm.book = response.data;
+		vm.bookCopy = angular.copy(vm.book); //Makes a copy of vm.books and sets it to booksCopy
```

Comment:
nice use of angular functions! you could also look at Object.assign for non-Angular apps :)